packages/material-ui/src/ClickAwayListener/ClickAwayListener.test.js

```diff
@@ -45,7 +45,7 @@ describe('<ClickAwayListener />', () => {
         </ClickAwayListener>,
       );
 
-      const event = fireBodyMouseEvent('mouseup');
+      const event = fireBodyMouseEvent('onClick');
```

Comment:
```suggestion
      const event = fireBodyMouseEvent('click');
```

All of `fireBodyMouseEvent('onClick')`'s should be changed.